### PR TITLE
Add Make recipe to allow for building and releasing

### PR DESCRIPTION
**Pull request by RyanMacG**
While we make a decision about a long term CD system this will allow us
do deployment from  local machine and push the new package to lambda

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
 
 services:
   - docker
 
 before_install:
   - docker-compose up -d
 
+after_success:
+  - pip install --no-cache-dir -r requirements.txt -t ./
+  - zip -r deletion-payload.zip
```
Comment (emileswarts) [inline .travis.yml]: Zip file will receive no contents here?

Comment (emileswarts) [reply .travis.yml]: Scrap that, we need to delete this

**Comment by adrianclay** [comment]
Looks good, an update to the README would be useful.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,25 @@
...
+## Testing
+
+To test the application we use pytest this can be run locally if you have mysql running and have imported the schema otherwise you can run;
+
+```shell
+$ make serve
+$ make test
+```
+
+This will start the database and application containers and import the database schema and then run the tests.
```
Comment (adrianclay) [inline README.md]: Mention requiring docker

Comment (RyanMacG) [reply README.md]: Done

**Pull request merged**